packages/Dockerfile

```diff
@@ -8,6 +8,9 @@ ARG SALT_VERSION
 ADD yum_repositories/*.repo /etc/yum.repos.d/
 RUN sed -i s/@SALT_VERSION@/$SALT_VERSION/ /etc/yum.repos.d/saltstack.repo
 
+# FIXME: CentOS Vault is broken...
```

Comment:
We probably want an issue (which explains the context & so on) to track that (build succeed so the fix works).

Reply:
Hum, but it breaks the bootstrap :sweat_smile:

Reply:
Ok, seems like the issues resolved itself so no need for a workaround on our side.

Reply:
I'll remove the commit